Automattic/sensei
# Generic answer feedback

Fixes #5370 (partially)

### Changes proposed in this Pull Request

*Add a feedback option for multi and single line questions.

### Testing instructions

*Add a multi or single line question and enter feedback.
*Feed back should display on relevant field on front end


### Screenshot / Video

<img width="881" alt="image" src="https://user-images.githubusercontent.com/4920132/217536366-96e57748-589c-4589-82f5-5b17349fdbe1.png">
